core/preferences/src/main/proto/settings.proto

```diff
@@ -111,11 +111,12 @@ message Settings {
     ClockWidgetLayout layout = 1;
     enum ClockStyle {
       DigitalClock1 = 0;
-      DigitalClock2 = 1;
-      OrbitClock = 5;
-      BinaryClock = 2;
-      AnalogClock = 3;
-      EmptyClock = 4;
+      OneClock = 1;
```

Comment:
Don't change the enum ordinals, it will mess up users' current settings. Just assign thr next higher number. Order matters as well because this is the order they appear in settings (which is the reason that this list is seemingly unordered)